### ISSUE TITLE
Add nlihc_id using multiple addresses

### ISSUE THREAD

**Issue by NealHumphrey** [codefordc/housing-insights#507]
We now have a one-to-many address table for projects. Linked data (TOPA notices, tax assessment, etc.) now need to have nlihc_id added to their records using all available mar ids instead of just the one on the project table.

**Comment by NealHumphrey** [comment]
@ptatian In the `Rcasd` files you've shared, which contain the TOPA notices, you've already done some geocoding of the records. Can you help clarify what an individual record is in this data set and how best to link this to the projects in our database?

Digging into a few examples, it looks like there is one row for each unique combination of address+notice. If one notice has multiple addresses, the `Nidc_rcasd_id` will be duplicated for each address. If one address has multiple notices, there will be a separate record for each `Nidc_rcasd_id`.  I'm assuming you've taken the original source data, which had one notice per row, and split each of those rows into multiple rows if multiple addresses were included. Finally, if the address range was written differently in two different TOPA notices the resulting multiple rows could be different. 

Is that all accurate? Is there anything I missed?

The case of "2301 Good Hope Court SE" in the address column appears to bear this out - it shows up in two TOPA notices (`2017-00012` and `2017-00155`); In the first topa notice the Orig_address is `2301-2323 Good Hope Court SE`, resulting in 12 unique addresses,  and the second is `2301-2327 Good Hope Court SE` resulting in 14 unique addresses.

**Comment by ptatian** [comment]
Yes, you are correct in all your assumptions. There is one unique record for each combination of address and notice. We parse out the address ranges into individual addresses. We don't make any real attempt to clean the address ranges -- we take them "as is."
